grails-app/views/_menu/_navbar.gsp

```diff
@@ -58,19 +58,12 @@
                         </g:link>
                     </li>
                     <li class="${controllerName.equals('aggregation') ? 'active' : ''}" id="aggregationMainMenu">
-                        <g:link controller="aggregation" action="show"
+                        <g:link controller="aggregationLegacy" action="show"
```

Comment:
The navigation bar links to the legacy page of the aggregation chart. Is this the desired behavior?